src/Pages/Contributors.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+
+const ContributorsPage = () => {
+  const { repoName } = useParams();
+  const [contributors, setContributors] = useState([]);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    // Scroll to top when the component mounts
+    window.scrollTo(0, 0);
+
+    const fetchContributors = async () => {
+      try {
+        const response = await fetch(`/api/projects/contributors/${repoName}`);
+        if (!response.ok) {
+          throw new Error("Server was unable to reach...");
+        }
+        const data = await response.json();
+
+        if (data.length > 0) {
+          const contributorsWithPics = await Promise.all(
```

Comment:
no need to fetch individual users from github api, we're already getting the data from `/api/projects/contributors/${repoName}`, just do `setContributors(data);`